src/js/mixin/modal.js

```
@@ -86,168 +116,175 @@ export default {
-            }
+                const handlers = [
+                    this.overlay && preventBackgroundFocus(this),
+                    this.overlay && preventBackgroundScroll(this.$el),
+                    this.bgClose && listenForBackgroundClose(this),
+                    this.escClose && listenForEscClose(this),
+                ];
 
+                once(
+                    this.$el,
+                    'hidden',
+                    () => handlers.forEach((handler) => handler && handler()),
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)